### DISCUSSION TITLE
Can content scale?

### DISCUSSION THREAD

**Discussion by psads-git** [cetz-package/cetz#675] [Q&A] [answered]
I am using the following code to insert a scaled `typst` math symbol into a circle. The `scale` function appears to have no effect at all on `content`. Is there any way of circumvent that?

```
#import "@preview/cetz:0.2.2"

#cetz.canvas({
  import cetz.draw: *

  scale(150%)
  circle((0,0), name: "plus1")

  scale(250%) // it has no effect
  content("plus1.center", $plus$)

})

#scale(275%, [$plus$])
```

**Comment by johannes-wolf** [comment, marked as answer]
No, this is not possible right now. Cetz uses a transformation matrix to transform coordinates, but Typst's provides no function to pass such a matrix to transform `Content`. You have to use Typst's native `scale` function inside the `content` call.

**Reply by psads-git** [reply]
Thanks, @johannes-wolf. I had already tried that, but got compiling errors with 

```
content("plus1.center", scale(275%, [$plus$]))
```

Maybe `cetz` `scale` function is overriding `typst` `scale` function.

**Reply by johannes-wolf** [reply]
> Thanks, @johannes-wolf. I had already tried that, but got compiling errors with
> 
> ```
> content("plus1.center", scale(275%, [$plus$]))
> ```
> 
> Maybe `cetz` `scale` function is overriding `typst` `scale` function.

It is, you have to store `scale` in a variable before `import draw: *`. With Typst >= 12 you can access native functions via some prefix, I think.
